openapi/paths/subscriptions@{id}@upcoming-invoices.yaml

```
@@ -11,8 +11,7 @@ get:
   description: >
     Retrieve an upcoming invoice from the specified subscription order.
 
-    The endpoint is temporary before upcoming invoices get a complete
-    integration.
+    This endpoint is deprecated and will soon be removed, use [Retrieve an upcoming invoice](../../Orders/operation/GetSubscriptionUpcomingInvoice)
```

Missing a period (this was causing the linter to fail)
```suggestion
    This endpoint is deprecated and will soon be removed, use [Retrieve an upcoming invoice](../../Orders/operation/GetSubscriptionUpcomingInvoice).
```